src/Dependencies/Microsoft.DiaSymReader/Microsoft.DiaSymReader.nuspec

```diff
@@ -5,10 +5,14 @@
     <summary>Microsoft DiaSymReader interop interfaces and utilities</summary>
     <description>
       Microsoft DiaSymReader interop interfaces and utilities.
-Supported Platforms:
-- .NET Framework 2.0
-- .NET Framework 4.5
     </description>  
+    <dependencies>
+      <group targetFramework="netstandard1.1">
+        <dependency id="System.Runtime" version="4.0.21-rc2-23910" />
```

Comment:
Latest versions are 4.1.0-rc2-23910 for both System.Runtime and System.Runtime.InteropServices.